meshes/meshops.pb.go

```diff
@@ -1,6 +1,8 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // source: meshops.proto
 
+//lint:file-ignore SA1019 Ignore the file until Google's protocol buffer supports gRPC Service Definitions. For more info see : https://github.com/layer5io/meshery/pull/848#issuecomment-613965380
```

Comment:
Also, this change is already done in a PR #848, please revert it.

Reply:
I think this is my mistake... I will sync my branch

Reply:
Same issue!

Reply:
should I remove this line no4  from my pr ??